### DISCUSSION TITLE
Power off rAudio remotely

### DISCUSSION THREAD

**Discussion by TinkeringB** [rern/rAudio#1220] [Tips] [answered]
derived from https://github.com/rern/rAudio-1/discussions/1158#discussion-4687702 (thanks!)

Am using Domoticz to switch on and off the various home-office equipment, and after recent experience of a corrupt SD card found the following to gracefully shutdown rAudio using Curl:
`curl -d "cmd=sh&sh%5B%5D=cmd.sh&sh%5B%5D=power&sh%5B%5D=off&sh%5B%5D=" -X POST http://rAudio/cmd.php`

and alternatively using Domoticz dzvents  openURL command:

```
domoticz.openURL({
	url = 'http://rAudio/cmd.php',
	method = 'POST',
	postData ='cmd=sh&sh%5B%5D=cmd.sh&sh%5B%5D=power&sh%5B%5D=off&sh%5B%5D=',
})
```

**Comment by TinkeringB** [comment]
Not working anymore after update rAudio 20230523

@rern  is there an alternative url I can call (from my domotica software) to shutdown the pi before switching off mains power?

**Comment by rern** [comment, marked as answer]
Try:
`curl -d "cmd=bash&filesh=power.sh" -X POST http://rAudio/cmd.php`
